node_package/src/wrapServerComponentRenderer/client.tsx

```
@@ -3,11 +3,12 @@
 import {
   ReactComponentOrRenderFunction,
   RenderFunction,
-  assertRailsContextWithComponentSpecificMetadata,
+  assertRailsContextWithServerStreamingCapabilities,
```

ESLint

[error] 'assertRailsContextWithServerStreamingCapabilities' is defined but never used.

(@typescript-eslint/no-unused-vars)